CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.21.0)
+cmake_minimum_required(VERSION 3.18.2)
```

Comment:
I found this to work for me locally, and as it's a minimum version I don't see any harm in lowering it.